src/azure-cli-core/azure/cli/core/extension/__init__.py

```diff
@@ -330,7 +330,7 @@ def get_extensions(ext_type=None):
     elif not isinstance(ext_type, list):
         ext_type = [ext_type]
     for t in ext_type:
-        extensions.extend([ext for ext in t.get_all()])
+        extensions.extend(t.get_all())
```

Comment:
`extend` accepts an `iterable`, so no need to convert `t.get_all()` to a list first.

- https://docs.python.org/3/library/stdtypes.html#mutable-sequence-types
- https://docs.python.org/3/tutorial/datastructures.html#more-on-lists